buddybuild_postbuild.sh

```diff
@@ -1,2 +1,24 @@
 #!/usr/bin/env bash
+
+echo "listing build dir in postbuild"
+echo $BUDDYBUILD_PRODUCT_DIR
+ls $BUDDYBUILD_PRODUCT_DIR
+
 bash <(curl -s https://codecov.io/bash)
+
+test_runner="XCUITests-Runner"
+branch_name=`git rev-parse --abbrev-ref HEAD`
+
+if [ "$BUDDYBUILD_SCHEME" = "Fennec"  ] && [ $branch_name == "master" ]; then
```

Comment:
Do we want to test this upload prior to merge and comment out master?

Reply:
let's do that and uncomment master branch check before we merge.  Nimbledroid people are making fixes on their side, and we probably need to send in a few builds.